src/styles/blocks/header.scss

```
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 0.3s ease;
```

"ease" is the default property
```suggestion
    transition: color 0.3s;
```